src/values/semrush/semrush-token.php

```diff
@@ -0,0 +1,153 @@
+<?php namespace Yoast\WP\SEO\Values\SEMrush;
+
+use League\OAuth2\Client\Token\AccessTokenInterface;
+use Yoast\WP\SEO\Exceptions\SEMrush\SEMrush_Empty_Token_Property_Exception;
+
+/**
+ * Class SEMrush_Token
+ * @package Yoast\WP\SEO\Values\SEMrush
+ */
+class SEMrush_Token {
```

Comment:
Not entirely convinced of the added value of this class over simply an associative array. It feels like a lot of added code that essentially only adds the `has_expired` function as actual logic.

Reply:
Possibly simply making all properties public and removing the other getters would help cut down on this.